src/app/logged-in-banner/logged-in-banner.component.scss

```diff
@@ -0,0 +1,24 @@
+.flex-container {
+  width: 100vw;
+  background: url('../../assets/images/dockstore/ship.png') no-repeat, url('../../assets/images/dockstore/truck.png') no-repeat, #21335b;
+  background-position: 45% 100%, 70% 100%;
+}
+
+.stacked-button {
+  margin: 0.25rem 0rem;
+  font-size: 16px;
+}
+
+.landing-search {
```

Comment:
If this is identical to the home.component.scss, move it to the style.scss and use that instead